src/AspNetCore.Mvc.HttpActionResults.ClientError/RequestedRangeNotSatisfiableResult.cs

```diff
@@ -43,6 +45,11 @@ public override Task ExecuteResultAsync(ActionContext context)
                 throw new ArgumentNullException(nameof(context));
             }
 
+            if (context.HttpContext.Response.Headers.Contains(new KeyValuePair<string, StringValues>(HeaderNames.ContentType, new StringValues(Resources.MultipartByterangesContentType))))
+            {
+                throw new OperationCanceledException(Resources.MustNotUseMultipartContentType);
```

Comment:
Remove the resources. We are not using them anywhere in the rest of code so it should stay consistent. It is not worthy to have a resource file for a single message. Additionally, we are not going to translate it anytime soon.